spec/behavior/ssl/ssl-connection.spec.js

```diff
@@ -73,7 +74,7 @@ describe('AMQP Connection', function () {
 
     before(function (done) {
       sandbox = sinon.createSandbox();
-      sslPathSettings.map((settingName) => {
+      sslPathSettings.forEach((settingName) => {
```

Comment:
Here I opted to use the native `forEach` instead of the lodash `each` because we were using the native version of `map`. Thoughts?

There's a few more like this marked with comments.

Reply:
I think this is a good change given we don't use any "return" from the `map` anyways